packages/webdriver/src/utils.js

```diff
@@ -218,13 +218,18 @@ export function isChrome (caps) {
  */
 export function isMobile (caps) {
     return Boolean(
-        (typeof caps['appium-version'] !== 'undefined') ||
+        (typeof caps['appium-version'] !== 'undefined') || (typeof caps['appiumVersion'] !== 'undefined') ||
         (typeof caps['device-type'] !== 'undefined') || (typeof caps['deviceType'] !== 'undefined') ||
         (typeof caps['device-orientation'] !== 'undefined') || (typeof caps['deviceOrientation'] !== 'undefined') ||
         (typeof caps.deviceName !== 'undefined') ||
+        // experitest returns actual device capabilities in the form "device.name" | "device.model" etc.
```

Comment:
I would not like to include experitest specific capabilties, only caps that are defined by [Appium](http://appium.io/docs/en/writing-running-appium/caps/)

Reply:
Do you mean I should remove all device. XYZ capabilities and just check for appiumVersion?